index.js

```diff
@@ -0,0 +1,98 @@
+const express = require("express");
+const configMiddleware = require("./config/middleware");
+const helper = require("./data/helper");
+
+// Create server
+const server = express();
+const PORT = 3000;
+
+// Middleware
+configMiddleware(server);
+
+// Start server
+let date = Date();
+
+server.get("/", (req, res) => {
+  res.send(" ✅ Getting Things Done™️ ✅ ");
+});
+
+// Project endpoints
+server.get("/api/projects", (req, res) => {
+  helper
+    .getProjects()
+    .then(projects => {
+      res.status(200).json(projects);
+    })
+    .catch(err => {
+      res.status(500).json({ message: "Could not load projects." });
```

Comment:
Why pass `err` as a parameter when you're not using it? What might be preferred (if you don't have a good reason) is empty parentheses.